corehq/apps/app_manager/static/app_manager/js/releases/app_view_release_manager.js

```diff
@@ -77,6 +77,32 @@ hqDefine("app_manager/js/releases/app_view_release_manager", function () {
     if (el.length) {
         el.koApplyBindings(releasesMain);
         _.defer(function () { releasesMain.goToPage(1); });
+
+        var releaseControlEl = $('#release-control');
+        if (releaseControlEl.length) {
+            releasesMain.showReleaseControl(false);
```

Comment:
From the other comment, if we are keeping `releasesEnabled` false by default now, this would not be needed.